x-pack/plugins/maps/public/components/metrics_editor/metric_editor.tsx

```diff
@@ -121,6 +145,29 @@ export function MetricEditor({
     );
   }
 
+  let percentileSelect;
+  if (metric.type === AGG_TYPE.PERCENTILE) {
+    percentileSelect = (
+      <EuiFormRow
+        label={i18n.translate('xpack.maps.metricsEditor.selectPercentileLabel', {
+          defaultMessage: 'Percentile',
+        })}
+        display="columnCompressed"
+      >
+        <EuiRange
```

Comment:
How about adding `showInput` parameter so you can enter number by typing. This will make setting an exact percentile much easier.

Reply:
discussed offline, just using number input instead